src/pate-chinois.md

```diff
@@ -0,0 +1,56 @@
+# Pâté Chinois (Quebec Style Shepherd’s Pie)
+
```

Comment:
Missing image.

Reply:
I personally don't have any pictures of the meal yet, any light image for Google would do?

Reply:
Luke doesn't want google images on this site... only images that have been taken by someone that has actually made the dish.

Reply:
Found one I took a while ago, it's not the prettiest thing to look at haha